website/client/components/static/home.vue

```diff
@@ -1,607 +1,458 @@
+      display: block;
+      text-align: center;
+      overflow: hidden;
+      white-space: nowrap;
+      margin-top: 1.5em;
+      margin-bottom: 1.5em;
+    }
+
+    .strike > span {
+      position: relative;
+      display: inline-block;
+    }
+
+    .strike > span:before,
+    .strike > span:after {
+      content: "";
+      position: absolute;
+      top: 50%;
+      width: 9999px;
+      height: 1px;
+      background: #fff;
+    }
+
+    .strike > span:before {
+      right: 100%;
+      margin-right: 15px;
     }
 
-    .bg-success {
-      background-color: #dff0d8 !important;
+    .strike > span:after {
+      left: 100%;
+      margin-left: 15px;
+    }
+
+    input {
+      margin-bottom: 1em;
+      border-radius: 2px;
+      background-color: #432874;
+      border-color: #432874;
+    }
+
+    button.sign-up {
+      width: 100%;
+      height: 48px;
+      color: #fff;
+      border: none;
+      border-radius: 2px;
+      background-color: #2995cd;
+      font-size: 16px;
     }
   }
 
-  #intro-text {
-    .lead {
-      padding: 2em;
+  #gamify-life {
+    text-align: center;
+
+    img {
+      margin-top: 1em;
+      margin-bottom: 1.5em;
+    }
+
+    .track-habits {
+      margin-bottom: 2.5em;
+    }
+
+    strong {
+      font-size: 20px;
     }
   }
 
-  #testimonial-carousel {
-    padding: 2em;
-    background-color: #d9edf7 !important;
+  #use-cases {
+    strong {
+      font-size: 20px;
+    }
 
-    h2 {
+    img {
+      display: block;
+      height: 200px;
+      width: 200px;
+      margin: 0 auto;
       margin-top: 2em;
-      margin-bottom: 3em;
+      margin-bottom: 2em;
     }
+  }
 
-    .item {
-      display: none;
+  #level-up-anywhere {
+    .app {
+      display: inline-block;
+      width: 135px;
+      margin-right: .5em;
     }
 
-    .item.active {
-      display: block;
+    .iphones {
+      width: 436px
+    }
+
+    .text-column {
+      padding-top: 9em;
     }
+  }
 
-    #quote-carousel {
-      margin-top: 7em;
+  #call-to-action {
+    .row {
+      margin-top: 1em;
+      margin-bottom: 1em;
+    }
 
-      .user-selector {
-        margin-top: 2em;
-      }
+    h3 {
+      font-size: 32px;
+    }
 
-      .user-icon {
-        display: inline-block;
-        vertical-align: bottom;
+    .btn-primary {
+      width: 411px;
+      height: 48px;
+      border-radius: 2px;
+      background-color: #9a62ff;
+      box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.24), 0 1px 4px 0 rgba(26, 24, 29, 0.16);
+      margin-bottom: 5em;
+    }
 
-        img {
-          width: 80px;
-        }
+    .container.featured {
+      padding-bottom: 0;
+    }
+
+    .container-fluid.featured {
+      padding-bottom: 5em;
+    }
+
+    .featured .row {
+      margin-top: 0;
+    }
+
+    .featured {
+      text-align: center;
+
+      img {
+        max-height: 30px;
+        max-width: 120px;
+        vertical-align: middle;
+        margin: 20px;
+        -moz-filter: brightness(0) invert(1);
```

Comment:
we're using autoprefixer so you don't have to add browser prefixes, they're added automatically by webpack